### ISSUE TITLE
Tests?

### ISSUE THREAD

**Issue by stevemao** [kevva/github-remove-forks#3]
https://github.com/mikedeboer/node-github has really great tests :D

**Comment by kevva** [comment]
I just get 404 when trying to run this atm. Maybe GitHub has changed something?

**Comment by stevemao** [comment]
Hm... Sorry, I don't remember if I have run it. But I read through the test files. Basically in the `before` hook you can just create some forks programmatically and test if you can still get the fork repos after you run your own module.

**Comment by kevva** [comment]
Yeah, I have tests ready, but it seems like something in the API is broken.
